s4e-web/src/app/views/map-view/product-picker/product-picker.component.ts

```diff
@@ -2,7 +2,7 @@ import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
 import {ProductType} from '../state/product-type/product-type.model';
 
 @Component({
-  selector: 's4e-product-picker',
+  selector: '[s4e-product-picker]',
```

Comment:
Better leave the tag as the selector. The other components do it that way and I don't see a reason to do it differently here.
Also: [Angular style guide](https://angular.io/guide/styleguide#style-05-03)